### PR TITLE
[FIX][14.0] hr_holidays: missing Time Off Description

### PR DESCRIPTION
**Pull request by royle-vietnam**
Steps:
1. Create a time off with description
2. Run migration
3. Missing Description

Solution: Move name to private_name on hr_leave to avoid missing Time Off Description

### CODE REVIEW

**Comment by MiquelRForgeFlow** [comment]
Then update https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_scripts/scripts/hr_holidays/14.0.1.5/upgrade_analysis_work.txt#L6

**Comment by pedrobaeza** [comment]
And why doing it in pre-migration?

**Comment by royle-vietnam** [comment]
> And why doing it in pre-migration?

Field 'name' on hr.leave has been changed to compute and no store

**Comment by pedrobaeza** [comment]
Then why don't you simply rename the column?

**Comment by royle-vietnam** [comment]
> Then why don't you simply rename the column?

Ok, good idea, thank you

**Comment by royle-vietnam** [comment]
> Then update https://github.com/OCA/OpenUpgrade/blob/14.0/openupgrade_scripts/scripts/hr_holidays/14.0.1.5/upgrade_analysis_work.txt#L6

Done, thank you

**Comment by royle-vietnam** [comment]
> Then why don't you simply rename the column?

Please review, thank you

File: openupgrade_scripts/scripts/hr_holidays/14.0.1.5/pre-migration.py [hunk 1]
```diff
@@ -4,11 +4,13 @@
...
 
 _column_copies = {
-    "hr_leave_type": [("validation_type", "allocation_validation_type", "varchar")]
+    "hr_leave_type": [("validation_type", "allocation_validation_type", "varchar")],
+    "hr_leave": [("name", "private_name", "varchar")],
 }
 
 _field_renames = [
     ("hr.leave.type", "hr_leave_type", "validation_type", "leave_validation_type"),
+    ("hr.leave", "hr_leave", "name", "private_name"),
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/scripts/hr_holidays/14.0.1.5/pre-migration.py]:
```suggestion
    ("hr.leave", "hr_leave", "name", "private_name"),
```
do the same with hr.leave.allocation

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: openupgrade_scripts/scripts/hr_holidays/14.0.1.5/pre-migration.py (hunk 1 above)
Comment (royle-vietnam) [reply openupgrade_scripts/scripts/hr_holidays/14.0.1.5/pre-migration.py]: Done, thank you

File: openupgrade_scripts/scripts/hr_holidays/14.0.1.5/upgrade_analysis_work.txt [hunk 2]
```diff
@@ -37,7 +41,11 @@ hr_holidays  / hr.leave.allocation      / interval_number (integer)     : now a
 hr_holidays  / hr.leave.allocation      / interval_unit (selection)     : now a function
 hr_holidays  / hr.leave.allocation      / manager_id (many2one)         : now a function
 hr_holidays  / hr.leave.allocation      / mode_company_id (many2one)    : now a function
+# NOTHING TO DO
+
 hr_holidays  / hr.leave.allocation      / name (char)                   : not stored anymore
+# DONE: rename field to private_name
```
Comment (pedrobaeza) [inline openupgrade_scripts/scripts/hr_holidays/14.0.1.5/upgrade_analysis_work.txt]: Put together both lines and use the text: `DONE: pre-migration: rename name > private_name`

Comment (royle-vietnam) [reply openupgrade_scripts/scripts/hr_holidays/14.0.1.5/upgrade_analysis_work.txt]: Done

Comment (pedrobaeza) [reply openupgrade_scripts/scripts/hr_holidays/14.0.1.5/upgrade_analysis_work.txt]: This is not what I said. Move this line to be together with the previous change and only one comment line for both:
```
hr_holidays  / hr.leave                 / private_name (char)           : NEW
hr_holidays  / hr.leave                 / name (char)                   : not stored anymore
# DONE: pre-migration: rename name > private_name
```
(and the same for the other model)

Comment (royle-vietnam) [reply openupgrade_scripts/scripts/hr_holidays/14.0.1.5/upgrade_analysis_work.txt]: I understand, done

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@MiquelRForgeFlow your turn

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**